storybook/preview.tsx

```diff
@@ -82,6 +59,8 @@ export const globalTypes = {
   },
 }
 
+export const themeManager = new ThemeManager(defaultTheme)
```

Comment:
Just wondering why you are exporting this new ThemeManager, as it's not used anywhere? Is that for future use?

Reply:
Rip, I probably could have not exported that. The only place where the `themeManager` is useful is for switching the active theme.